components/progressbar/stories/meter.template.js

```diff
@@ -67,5 +67,27 @@ export const MeterGroup = (args, context) => html`
 				}, context)}
 			</div>
 		`)}
+
+		<!-- The gradient story below supports linear-gradients used by Express. For use cases that require a custom linear-gradient for any --mod-*-{fill} properties, set those custom properties in CSS. -->
+		 <div>
+			${Typography({
+				semantics: "heading",
+				size: "s",
+				content: ["Gradient support"],
+				customClasses: ["chromatic-ignore"],
+			}, context)}
+			<div style=${styleMap({
+				"display": window.isChromatic() ? "flex" : "none",
+				"flex-direction": "column",
+				"align-items": "flex-start",
+				"gap": "32px",
+			})}>
+				${Template({
+					...args,
+					trackFill: "linear-gradient(to right, hotpink, orange)",
+					progressBarFill: "linear-gradient(to left, teal, purple)",
+				}, context)}
+			</div>
+		</div>
```

Comment:
Could what this section is displaying be achieved by adding an object with:
```
{
	heading: "Gradient support",
	trackFill: "linear-gradient(to right, hotpink, orange)",
	progressBarFill: "linear-gradient(to left, teal, purple)",
}
```
to the existing `map` function above this instead?

Reply:
ooooo yes! great idea once again! https://github.com/adobe/spectrum-css/pull/2929/commits/f5db3f3ed0f150799182933c98ec349db65652bb